cpp/modmesh/buffer/pymod/wrap_SimpleArrayPlex.cpp

```diff
@@ -125,13 +125,15 @@ static void verify_python_value_datatype(pybind11::object const & value, DataTyp
         {
             throw pybind11::type_error("Data type mismatch, expected Complex64");
         }
+        break;
```

Comment:
We do not have unit tests covering this, but it is probably too detail for testing.